### PR TITLE
Add Borinot robot

### PR DESCRIPTION
**Pull request by stephane-caron**
Opening this PR after [Borinot: an agile torque-controlled robot for hybrid flying and contact loco-manipulation (workshop version)](https://arxiv.org/abs/2305.01423).

Info we should complete to [add the robot to the list](https://github.com/tasts-robots/awesome-open-source-robots/blob/main/CONTRIBUTING.md#adding-a-robot-to-the-list):

- License?
- Link to build instructions?
- Link to motion control software?

### CODE REVIEW

**Comment by stephane-caron** [comment]
@joansola Starting this draft PR to reference Borinot. Feel free to share the links/info when they become available. :smiley:

**Comment by joansola** [comment]
Hi Stephane. I-ll fwd this PR to my colleagues working on Borinot. thanks!

**Comment by joansola** [comment]
We'll have the assembly instructions and repos ready and open very soon!

**Comment by PepMS** [comment]
Hi, thanks for the interest @stephane-caron 
 
The preprint version of the paper presenting Borinot is already available at: https://arxiv.org/abs/2307.14686

Also, you can find a very premature version of related documentation at: www.iri.upc.edu/borinot
We will keep updating this documentation in the upcoming weeks. We are still validating the assembly instructions and other procedures in our lab.

We'll make sure to keep you posted.

PS: @bacmel, would you keep an eye on this PR? Thanks! :)

**Comment by stephane-caron** [comment]
Cool, I'm happy to see things shaping up :smiley: I've rebased the branch on `main` and fixed the linter error. Feel free to chime in here when the docs and software are ready.

**Comment by Bacmel** [comment]
Hello @stephane-caron!
Thank you for your interest in this project! (I know I'm late, sorry haha)

We now have a proper repo for this project where you can find "everything" about Borinot: [www.iri.upc.edu/borinot](http://www.iri.upc.edu/borinot) (this link is associated with the repo in question, so it's best to use it).

By everything, I mean:
- Licence (BSD 3 clause licence)
- The build instruction
- The Bill of Material
- Software configuration
- 3D files (for printing)

There are still a few improvements to be made and I think it would be a good idea to get some feedback first before embarking on a complete rebuild of the project ahah.

So here it is:

---

Hi there! Here is a proposal to add the open source robot ``Borinot`` 🐝

* [x] I have checked that the build is open ([guidelines](https://github.com/tasts-robots/us/blob/main/CONTRIBUTING.md))
* [x] I have checked that the software to control the robot is open ([guidelines](https://github.com/tasts-robots/us/blob/main/CONTRIBUTING.md))

Relevant information about the robot:

* Type: (Unmanned Aerial Loco-Manipulator) 
* License: (BSD 3 clause licence)
* Build instructions: ([Borinot Hardware Assembly](https://github.com/hidro-iri/Borinot/blob/main/hardware/README.md))
* Model: (URDF)
* Software: ([PX4](https://github.com/PX4/PX4-Autopilot),[odri_control_interface](https://github.com/open-dynamic-robot-initiative/odri_control_interface))

Icing on the cake, here is a picture for the gallery:

Stunning photo of a wild Borinot

Robotically,
Hugo Duarte

File: README.md [hunk 1]
```diff
@@ -38,6 +39,12 @@ A curated list of awesome robots that are fully open source, both hardware and s
 
 \* CC-BY-NC-SA is not an open source license. See the [Open Source Definition](https://en.wikipedia.org/wiki/The_Open_Source_Definition).
 
+### Hybrid
+
+| Name  | Type          | License    | Build | Software |
+|-------|---------------|------------|-------|----------|
+| [Borinot](https://arxiv.org/abs/2305.01423) | Drone + Arm | ? | ? | ? |
```
Comment (stephane-caron) [inline README.md]:
```suggestion
| [Borinot](https://arxiv.org/abs/2305.01423) | Drone + Arm | BSD-3-Clause | [Instructions](https://github.com/hidro-iri/Borinot/blob/main/hardware/README.md) | [GitHub](https://github.com/hidro-iri/Borinot/blob/main/software/README.md) |
```

**Comment by stephane-caron** [comment]
Awesome, congrats for the work! I'm looking forward to seeing more wild Borinots roaming around :smiley:

**Pull request merged**